lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
```

Comment:
It looks like you might be missing a bin file to tie all of the code together and execute it.